### PR TITLE
ISSTA 2022 Distinguished Papers

### PR DESCRIPTION
**Pull request by tzimmermsr**
@raux: Please merge to the main branch and the live site.

### CODE REVIEW

File: awards/distinguishedPaperAward.html [hunk 1]
```diff
@@ -39,6 +39,12 @@ <h3>How to Prepare ACM SIGSOFT Distinguished Papers Award Certificates</h3>
 <a name="winners"></a>
 <h2>Winners</h2>
 
+<h3>ACM SIGSOFT International Symposium on Software Testing and Analysis (ISSTA 2022)<br>
```
Comment (raux) [inline awards/distinguishedPaperAward.html:42]: LGTM

**Comment by raux** [comment]
Please make sure that content is accurate. 

Raula

**Comment by tzimmermsr** [comment]
I had the wrong end tag for the h3-title. I submitted a new pull request to fix it.

**Comment by raux** [comment]
@tzimmermsr no worries, I actually fixed it and added the date and venue just to be consistent with the prior entries.

Please confirm this is correct.

Raula

**Pull request merged**